### ISSUE TITLE
Table formatting: Should not make the lines too long

### ISSUE THREAD

**Issue by rugk** [slang800/atom-tidy-markdown#20]
E.g. when using a link in a table the lines and headers could get very long. I think there should be a kind of limit for that or a special handling of links in tables.

One thing I did to make the line shorter is to use the [link referring](https://daringfireball.net/projects/markdown/syntax#link) of markdown.

**Comment by notslang** [comment]
Yeah, I've noticed the same thing on some of my tables (even ones that don't have many links), and moving long links out of the markdown and into a set of refs at the bottom is definitely something I want to implement.
